docs/api/text/postag.rst

```diff
@@ -224,7 +224,7 @@ Libraries like SpaCy use tags from the Universal Dependencies (UD) framework. Th
 +-------------------+------------------------------------------+----------------------+--------------------------+
 
 Parsing raw text automatically
----------------------------
+------------------------------
```

Comment:
Note: I added this because I was getting a docs error message after we merged develop into this PR branch